package.json

```
@@ -24,5 +24,10 @@
   "dependencies": {
     "@stencil/core": "^2.0.1"
   },
+  "devDependencies": {
+    "@types/jest": "^26.0.24",
+    "jest": "^27.0.6",
```

Stencil doesn't support Jest 27 at this moment. It's something the team is working on, but with the latest version of Stencil that's publicly available, running `npm test` will fail:
```
npm t

> stencil-starter-project-name@0.0.1 test
> stencil test --spec --e2e

[22:08.4]  @stencil/core
[22:08.5]  v2.6.0 📟
[22:08.6]  testing e2e and spec files
[22:09.5]  build, stencil-starter-project-name, dev mode, started ...
[22:09.6]  transpile started ...
[22:11.4]  transpile finished in 1.85 s
[22:11.4]  copy started ...
[22:11.4]  generate custom elements bundle started ...
[22:11.4]  generate lazy started ...
[22:11.5]  copy finished (0 files) in 106 ms
[22:11.8]  generate custom elements bundle finished in 322 ms
[22:11.8]  generate lazy finished in 320 ms
[22:11.8]  build finished in 2.25 s

[22:25.7]  jest args: --e2e --spec --max-workers=8

[ ERROR ]  runJest: Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './build/cli/args' is not defined by
           "exports" in /Users/ryan/workspaces/stencil-component-starter/node_modules/jest-cli/package.json
```
Would you mind downgrading this to `jest@26.6.3`:
```suggestion
    "jest": "^26.6.3",
```